diff --git a/settings/ci.settings.php b/settings/ci.settings.php
index aa9f811..481b244 100644
--- a/settings/ci.settings.php
+++ b/settings/ci.settings.php
@@ -20,7 +20,7 @@ $settings['trusted_host_patterns'] = [
  */
 $databases['default']['default'] = [
   'database' => 'drupal',
-  'username' => 'drupal',
+  'username' => 'root',
   'password' => 'drupal',
   'host' => '127.0.0.1',
   'port' => '3306',
